.github/workflows/pull.yml

```diff
@@ -621,71 +621,87 @@ jobs:
           python torchchat.py remove stories15m
 
   test-mps:
-    uses: pytorch/test-infra/.github/workflows/macos_job.yml@main
```

Comment:
This test is very flaky because it fails whenever a machine with python 3.9 is allocated.  Change to always setup python 3.10.

Reply:
Thanks for fixing this test- it's been a thorn on our side for a while